qiskit_metal/qlibrary/qubits/star_qubit.py

```diff
@@ -0,0 +1,389 @@
+
+        ##################################################################
+        # Add geometry and Qpin connections
+        p_in = (p.pos_y, (p.pos_y + p.radius))
+        p_out = (p.pos_y, 1.25 * (p.pos_y + p.radius))
+        pins = draw.LineString([p_in, p_out])
+        pins1 = draw.rotate(pins, p.rotation1, origin=(p.pos_x, p.pos_y))
+        pins2 = draw.rotate(pins, p.rotation2, origin=(p.pos_x, p.pos_y))
+        pins3 = draw.rotate(pins, p.rotation3, origin=(p.pos_x, p.pos_y))
+        pins4 = draw.rotate(pins, p.rotation4, origin=(p.pos_x, p.pos_y))
+        pins5 = draw.rotate(pins, p.rotation5, origin=(p.pos_x, p.pos_y))
+
+        self.add_qgeometry('poly', {'circle1': total},
+                           subtract=p.subtract,
+                           helper=p.helper,
+                           layer=p.layer,
+                           chip=p.chip)
+        if (p.number_of_connectors) == 0:
+            self.add_qgeometry('poly', {'contact3': contact3},
+                               subtract=p.subtract,
+                               helper=p.helper,
+                               layer=p.layer,
+                               chip=p.chip)
+            self.add_pin('pin3', pins3.coords, width=0.01, input_as_norm=True)
+        elif (p.number_of_connectors) == 1:
+            self.add_qgeometry('poly', {'contact1': contact1},
+                               subtract=p.subtract,
+                               helper=p.helper,
+                               layer=p.layer,
+                               chip=p.chip)
+            self.add_qgeometry('poly', {'contact3': contact3},
+                               subtract=p.subtract,
+                               helper=p.helper,
+                               layer=p.layer,
+                               chip=p.chip)
+            # Add pin connections
+            self.add_pin('pin1', pins1.coords, width=0.01, input_as_norm=True)
+
+            self.add_pin('pin3', pins3.coords, width=0.01, input_as_norm=True)
+        elif (p.number_of_connectors) == 2:
+            self.add_qgeometry('poly', {'contact1': contact1},
+                               subtract=p.subtract,
+                               helper=p.helper,
+                               layer=p.layer,
+                               chip=p.chip)
+            self.add_qgeometry('poly', {'contact2': contact2},
+                               subtract=p.subtract,
+                               helper=p.helper,
+                               layer=p.layer,
+                               chip=p.chip)
+            self.add_qgeometry('poly', {'contact3': contact3},
+                               subtract=p.subtract,
+                               helper=p.helper,
+                               layer=p.layer,
+                               chip=p.chip)
+            # Add pin connections
+            self.add_pin('pin1', pins1.coords, width=0.01, input_as_norm=True)
```

Comment:
Is it ok to keep the width fixed? Would we not want to set this to a varaible such as the CPW width? For instance in the transom pads, i think we use that:
https://github.com/Qiskit/qiskit-metal/blob/1ee7b39eeeffe9b8ad1119b5d607c91a896696c1/qiskit_metal/qlibrary/qubits/transmon_pocket.py#L124


https://github.com/Qiskit/qiskit-metal/blob/1ee7b39eeeffe9b8ad1119b5d607c91a896696c1/qiskit_metal/qlibrary/qubits/transmon_pocket.py#L212

Reply:
@smukesh25 can you please consider this?